sui_programmability/examples/games/hero/sources/Hero.move

```diff
@@ -8,7 +8,7 @@ module HeroGame::Hero {
     use Sui::Event;
```

Comment:
Excellent! I think this means we can now move `Hero.move` directly into the `games/sources` directory instead of giving it its own lonely subdir.